frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/error/error_block_component_builder.dart

```diff
@@ -111,4 +103,44 @@ class _ErrorBlockComponentWidgetState extends State<ErrorBlockComponentWidget>
 
     return child;
   }
+
+  Widget _buildDesktopErrorBlock(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 12),
+      child: Wrap(
+        crossAxisAlignment: WrapCrossAlignment.center,
+        children: [
+          const HSpace(4),
+          FlowyText.regular(
+            LocaleKeys.document_errorBlock_theBlockIsNotSupported.tr(),
+          ),
+          const HSpace(4),
+          FlowyText.regular(
+            '(${LocaleKeys.document_errorBlock_clickToCopyTheBlockContent.tr()})',
+            color: Theme.of(context).hintColor,
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildMobileErrorBlock(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 12, horizontal: 4),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          FlowyText.regular(
+            LocaleKeys.document_errorBlock_theBlockIsNotSupported.tr(),
+          ),
+          const VSpace(6),
+          FlowyText.regular(
+            '(${LocaleKeys.document_errorBlock_clickToCopyTheBlockContent.tr()})',
```

Comment:
We could just add the parantheses to the localisation.

Reply:
I was thinking it would be a little bit redundant to add the parentheses to the translation. So I just added it here.